slack_pull_reminder.py

```diff
@@ -90,7 +124,9 @@ def send_to_slack(text):
 
 
 def cli():
-    lines = fetch_organization_pulls(ORGANIZATION)
+    org = get_organization(ORGANIZATION)
+    add_TEAMS_usernames_to_USERNAMES(org)
```

Comment:
I think it might be nicer if instead of mutating `USERNAMES` you do something like:

```python
selected_usernames = USERNAMES or fetch_team_usernames()
```

and then pass that on to `fetch_organization_pulls`